### DISCUSSION TITLE
Idea: Community Repo as forum + wiki x+ research notebook (digital garden / second brain)

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#169] [General] [answered]
<div align="center">
<h3>UPDATE: analysis and plan have moved to <a href="https://github.com/life-itself/community/blob/main/projects/digital-garden-2022.md">the digital garden 2022 project file</a><br />(became epic enough to have its own mini-project file here)</h3>
</div>

---

Idea: use this repository as notebook/wiki/forum and have that published online at e.g. forum.lifeitself.us or lifeitself.us/forum/

There are roughly two parts to this:

* A structure for adding content, discussions etc
* A way of publishing that online

### Notebook

When I have notes on a book I want to be able to publish them somewhere so that others (and myself) can find them

When I have quick ideas or notes (incl e.g. a screenshot) I want to be able to store (and publish) them somewhere so I can find and link to them later

When creating notes or ideas or questions I want to be able to discuss them with others so that they get get refined or answered

### Forum

When I have ideas or thoughts (relatively unformed) I want to share with others so that i can get input / feedback

When I want to learn more day to day about a community I want to have somewhere I can see what they are up to and thinking so that ...

When I am loosely associated

## Context

This community repository current houses a few things:

Git repo itself
* Issues
* Discussions

### Background:

* This repository was originally our (fairly inactive) "research" repository.
* We also used to have a forum powered by discourse.

## Analysis

New-fangled graph-oriented knowledge systems seem all the rage e.g. Roam and now a plethora of others Foam, Dendron, Obsidian, Athens Research etc etc (though seem incredibly similar to wikis ... but with block level linking ...)

Would be nice to use this and perhaps the zettelkasten methodology.

My recommendation would be Obsidian or Dendron I think. Dendron seems better than Foam.

**Comment by Liyubov** [comment]
Just a quick question regarding this folder on digital garden: https://github.com/life-itself/community/blob/main/projects/digital-garden-2022.md

In research folder I posted the information about some community analysis methods, which look at community at more theoretical level averaged over time period X. 
Idea of research are [here ](https://github.com/life-itself/community/tree/main/research)

The idea of their development are: 

- [ ] look into other existing theoretical research on this area 
- [ ] apply this methodolody for other communities to map them and compare with Life itself
- [ ] develop methodology for describing community trajectories for everyday life (and for more usable cases 

@rufuspollock do not know about the [digital garden](https://github.com/life-itself/community/blob/main/projects/digital-garden-2022.md) project, if it is meant to be a project of community itself or part of research?

**Comment by rufuspollock** [comment]
@Liyubov the digital garden is about creating infrastructure i.e. a community wiki/forum (aka digital garden) - not doing research itself.

**Comment by Liyubov** [comment]
i see. it seems that community wiki/forum would be great material for research part actually to analyze/look at. 
if there are any documents (wikis) which try to document them, indeed i would put them into research folder as examples to analyse community (as discussed in closed issue #115 )

**Comment by rufuspollock** [comment, marked as answer]
Ideation and discussion are now complete and this is now in progress. See tracking epic https://github.com/life-itself/community/issues/170 and ~~https://github.com/life-itself/community/blob/main/projects/digital-garden-2022.md~~ https://github.com/life-itself/community/blob/main/content/notes/projects/digital-garden-2022.md

**Reply by asimong** [reply]
link not working today 2023-01-21
